packages/amplify-sdk/src/amplify-sdk.js

```diff
@@ -137,10 +137,12 @@ export default class AmplifySDK {
 								}
 								return obj;
 							}, {}),
-						guid:         org.guid,
-						id:           org.org_id,
-						name:         org.name,
-						region:       org.region
+						guid:          org.guid,
+						id:            org.org_id,
+						name:          org.name,
+						region:        org.region,
+						subscriptions: org.subscriptions,
```

Comment:
**Question:** Should we do a null/undefined check on "subscriptions" since other functions assume this is always assigned an array?
```javascript
subscriptions: org.subscriptions || [],
```

Reply:
Fixed!